test/integration/graql/query/GraqlInsertIT.java

```diff
@@ -275,46 +151,11 @@ public void whenInsertingAResourceWithMultipleValues_Throw() {
         tx.execute(Graql.insert(varPattern));
     }
 
-    @Test
-    public void testErrorWhenSubRelation() {
-        exception.expect(IllegalArgumentException.class);
-        tx.execute(Graql.insert(
-                var().sub("has-genre").rel("genre-of-production", "x").rel("production-with-genre", "y"),
-                var("x").id("Godfather").isa("movie"),
-                var("y").id("comedy").isa("genre")
-        ));
-    }
-
     @Test
     public void testInsertRepeatType() {
         assertInsert(var("x").has("title", "WOW A TITLE").isa("movie").isa("movie"));
     }
 
-    @Test
-    public void testKeyCorrectUsage() throws InvalidKBException {
-        tx.execute(Graql.define(
-                type("a-new-type").sub("entity").key("a-new-resource-type"),
-                type("a-new-resource-type").sub(Graql.Token.Type.ATTRIBUTE).value(Graql.Token.ValueType.STRING)
-        ));
-
-        tx.execute(Graql.insert(var().isa("a-new-type").has("a-new-resource-type", "hello")));
-    }
-
-    @Test
-    public void whenInsertingAThingWithTwoKeyResources_Throw() throws InvalidKBException {
-        tx.execute(Graql.define(
-                type("a-new-type").sub("entity").key("a-new-attribute-type"),
-                type("a-new-attribute-type").sub(Graql.Token.Type.ATTRIBUTE).value(Graql.Token.ValueType.STRING)
-        ));
-
-        tx.execute(Graql.insert(
-                var().isa("a-new-type").has("a-new-attribute-type", "hello").has("a-new-attribute-type", "goodbye")
-        ));
-
-        exception.expect(InvalidKBException.class);
-        tx.commit();
-    }
-
     @Ignore // TODO: Un-ignore this when constraints are designed and implemented
     @Test
     public void testKeyUniqueValue() throws InvalidKBException {
```

Comment:
let's have a look at the intent of this test?

Reply:
As it turns out, it's just testing key uniqueness, which we have a BDD scenario for already, so this IT will be deleted.